bgapi/module.py

```diff
@@ -55,6 +63,43 @@ def get_services(self):
         self.parse_advertisement_data()
         return self.services
 
+    def get_ad_type_string(self, type):
```

Comment:
I've never seen a python look up table done this way, I like it, I'm going to start using this style.